campaign-launcher/interfaceV2/src/api/recordingApiClient.ts

```diff
@@ -174,8 +175,8 @@ export class RecordingApiClient {
     return response;
   }
 
-  async upsertExchangeApiKey(exchangeName: string, apiKey: string, secret: string): Promise<{ id: string }> {
-    const response = await this.post<{ id: string }>(`/exchange-api-keys/${exchangeName}`, {
+  async upsertExchangeApiKey(exchangeName: string, apiKey: string, secret: string): Promise<void> {
+    const response = await this.post<void>(`/exchange-api-keys/${exchangeName}`, {
```

Comment:
```suggestion
    await this.post<void>(`/exchange-api-keys/${exchangeName}`, {
```
and do not return anything since it's `void` now